steps/zscan-upload-plugin/1.0.1/step.yml

```diff
@@ -0,0 +1,88 @@
+title: zScan Upload Plugin
+summary: |
+  Bitrise plugin to upload to Zimperium zScan
+description: |
+  This plugin can be used to upload mobile applications to Zimperium (zScan) to be scanned for vulnerabilities. Using a plugin simplifies integrating mobile application security testing into CI/CD process and enables detection and remediation of vulnerabilities earlier in the application SDLC.
+website: https://github.com/Zimperium/zscan-plugin-bitrise
+source_code_url: https://github.com/Zimperium/zscan-plugin-bitrise
+support_url: https://github.com/Zimperium/zscan-plugin-bitrise/issues
+published_at: 2024-11-26T09:53:28.573255-06:00
+source:
+  git: https://github.com/Zimperium/zscan-plugin-bitrise.git
+  commit: 2e72a6c38ffe1b17fbee41e2b45c1dc6e550367d
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: jq
+  - name: curl
+  apt_get:
+  - name: jq
+  - name: curl
+is_skippable: true
+inputs:
+- endpoint: https://ziap.zimperium.com
+  opts:
+    description: |
+      This is the base URL of your console, e.g., `https://ziap.zimperium.com`..
+    is_expand: true
+    is_required: true
+    summary: Console base URL.
+    title: Endpoint
+    value_options: []
```

Comment:
This can be deleted, it only needs to be present when the input is an enum with predefined values. Same for all other inputs that define this.